code/io/timer.h

```diff
@@ -138,6 +138,9 @@ UI_TIMESTAMP ui_timestamp(int delta_ms);
 TIMESTAMP timestamp_delta(TIMESTAMP stamp, int delta_ms);
 UI_TIMESTAMP ui_timestamp_delta(UI_TIMESTAMP stamp, int delta_ms);
 
+int timestamp_get_delta(TIMESTAMP before, TIMESTAMP after);
+int ui_timestamp_get_delta(TIMESTAMP before, TIMESTAMP after);
```

Comment:
The parameters to the ui_ function need to be the UI_TIMESTAMP type.

(After 22.2 is out I'm going to merge the regular and ui_ functions using templates.  The types need to be distinct, but the functions can be the same.)